src/ffi/datetime.rs

```diff
@@ -445,12 +445,11 @@ pub static PyDateTimeAPI: _PyDateTimeAPI_impl = _PyDateTimeAPI_impl {
     inner: GILOnceCell::new(),
 };
 
-/// Safe wrapper around the Python C-API global `PyDateTime_TimeZone_UTC`. This follows a similar
+/// Wrapper around the Python C-API global `PyDateTime_TimeZone_UTC`. This follows a similar
```

Comment:
Is this not a safe wrapper now?

Reply:
Well, I made it less safe because it gives you `&'static *mut ffi::PyObject`, which is a pointer and thus less safe to use. The safe wrapper is now the `pyo3::types::timezone_utc()` function.